canvas_modules/common-canvas/src/common-properties/ui-conditions/conditions-utils.js

```diff
@@ -709,6 +709,15 @@ function _validateInput(propertyId, controller, control, showErrors) {
 				if (!controller.getErrorMessage(msgPropertyId, true, true, false)) {
 					errorSet = false;
 				}
+				// Before setting an error message for table cell, clear the error message for table (if any)
+				if (typeof msgPropertyId.row !== "undefined" || typeof msgPropertyId.col !== "undefined") {
```

Comment:
Is this still needed?

Reply:
Yes. This code is fixing the following repeated error messages issue - 
<img width="326" alt="Screenshot 2023-07-27 at 10 52 40 AM" src="https://github.com/elyra-ai/canvas/assets/25124000/e71471c0-6eed-4346-90d5-51341c637c0b">